src/components/footnote-group/footnote-group.scss

```diff
@@ -0,0 +1,31 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/theme';
+@use '../../mixins/ms';
+
+/// Footnote groups are used to group footnotes together in a list
+.c-footnote-group {
+  --spacing-footnotes: #{ms.step(2)};
+
+  border-top: size.$edge-medium solid color.$base-gray-lighter;
+  margin-top: var(--spacing-footnotes);
+  padding-top: var(--spacing-footnotes);
+
+  @include theme.styles(dark) {
+    border-color: color.$brand-primary-light;
+  }
+}
+
+.c-footnote-group__list > * + * {
+  margin-top: var(--spacing-footnotes);
+}
+
+/// Compact footnote groups have smaller text, smaller gaps, and no top
+/// border/padding
+.c-footnote-group--compact {
+  --spacing-footnotes: #{ms.step(1)};
+
+  border-top: none;
+  font-size: ms.step(-1);
+  padding-top: 0;
+}
```

Comment:
Since this is a modifier of `.c-footnote-group`, can we put it closer to that element (instead of following its descendants)?

Reply:
Yep 👍

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/b3434f1411d984dfc42d38bc8f979a7419d36a60